sqlserver azuredb=true gives segfault

### Relevant telegraf.conf:
```
[[inputs.sqlserver]]
  servers = ["Server=xxx.database.windows.net;Port=1433;Database=xxx;User Id=xxx;Password=xxx;app name=telegraf;log=1;"]
  query_version = 2
  azuredb = true
```

### System info:

Telegraf 1.9.4
AzureSQL Standard

### Steps to reproduce:

`telegraf -test -debug -config=/etc/telegraf/telegraf.conf`

### Expected behavior:

```
...
> sqlserver_performance,counter=Reduced\ Memory\ Grant\ Count,host=telegraf,instance=SeedingGroup,object=SQLServer:Workload\ Group\ Stats,sql_instance=xxx value=0 1550652485000000000
> sqlserver_performance,counter=Request\ Count,host=telegraf,instance=SeedingGroup,object=SQLServer:Workload\ Group\ Stats,sql_instance=xxx value=0 1550652485000000000
> sqlserver_performance,counter=Request\ Count,host=telegraf,instance=SloDTAGroup,object=SQLServer:Workload\ Group\ Stats,sql_instance=xxx value=0 1550652485000000000
....

```
### Actual behavior:

2019-02-20T08:44:40Z I! Starting Telegraf 1.9.4
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x142b968]

goroutine 30 [running]:
github.com/influxdata/telegraf/plugins/inputs/sqlserver.(*SQLServer).accRow(0xc0002e2b80, 0x22969b2, 0x263, 0x1, 0xc0003bc540, 0xc, 0xc, 0x252f720, 0xc000170820, 0x2500620, ...)
        /go/src/github.com/influxdata/telegraf/plugins/inputs/sqlserver/sqlserver.go:222 +0x738
github.com/influxdata/telegraf/plugins/inputs/sqlserver.(*SQLServer).gatherServer(0xc0002e2b80, 0xc000250121, 0x86, 0x22969b2, 0x263, 0x1, 0xc0003bc540, 0xc, 0xc, 0x252f720, ...)
        /go/src/github.com/influxdata/telegraf/plugins/inputs/sqlserver/sqlserver.go:178 +0x20c
github.com/influxdata/telegraf/plugins/inputs/sqlserver.(*SQLServer).Gather.func1(0xc00018b4b0, 0x252f720, 0xc000170820, 0xc0002e2b80, 0xc000250121, 0x86, 0x22969b2, 0x263, 0x1, 0x0, ...)
        /go/src/github.com/influxdata/telegraf/plugins/inputs/sqlserver/sqlserver.go:141 +0xb7
created by github.com/influxdata/telegraf/plugins/inputs/sqlserver.(*SQLServer).Gather
        /go/src/github.com/influxdata/telegraf/plugins/inputs/sqlserver/sqlserver.go:139 +0x1e8

shell returned 2

### Additional info:

Removing `azuredb = true` fixes behavior...

--- a/plugins/inputs/sqlserver/sqlserver.go
+++ b/plugins/inputs/sqlserver/sqlserver.go
@@ -5,11 +5,9 @@ import (
 	"sync"
 	"time"
 
+	_ "github.com/denisenkom/go-mssqldb" // go-mssqldb initialization
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/plugins/inputs"
-
-	// go-mssqldb initialization
-	_ "github.com/denisenkom/go-mssqldb"
 )
 
 // SQLServer struct
@@ -90,7 +88,7 @@ func initQueries(s *SQLServer) {
 
 	// If this is an AzureDB instance, grab some extra metrics
 	if s.AzureDB {
-		queries["AzureDB"] = Query{Script: sqlAzureDB, ResultByRow: true}
+		queries["AzureDB"] = Query{Script: sqlAzureDB, ResultByRow: false}
 	}
 
 	// Decide if we want to run version 1 or version 2 queries

